src/Components/StandardContainer/index.jsx

```
@@ -0,0 +1,51 @@
+// Components
+import { Stack } from "@mui/material";
+
+// Utils
+import { useTheme } from "@emotion/react";
+
+const Container = ({ children, direction, backgroundColor, sx }) => {
```

ESLint

[error] 'children' is missing in props validation

(react/prop-types)